server/tsconfig.test.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": "./tsconfig.json"

```

Comment:
Wait, does it still make sense to extend some config without overriding anything? I guess copying everything sure isn't an option, but… Well, I dunno. I guess this version is the best we can do? What if this file is deleted altogether?

Reply:
I think we should be able to delete this file safely.